scripts/ubuntu/hello-world-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
I think it's good to stay away from any "global" settings and just put this in front of any command that needs it.